src/main/java/pl/betoncraft/betonquest/events/JournalEvent.java

```diff
@@ -18,27 +18,43 @@
 import pl.betoncraft.betonquest.utils.Utils;
 
 import java.util.Date;
+import java.util.Locale;
 import java.util.logging.Level;
 
 /**
- * Adds the entry to player's journal
+ * Adds the entry to player's journal.
  */
 @SuppressWarnings("PMD.CommentRequired")
 public class JournalEvent extends QuestEvent {
 
     private final String name;
     private final boolean add;
 
+    @SuppressWarnings("PMD.CyclomaticComplexity")
     public JournalEvent(final Instruction instruction) throws InstructionParseException {
         super(instruction, false);
         staticness = true;
-        final String first = instruction.next();
-        if ("update".equalsIgnoreCase(first)) {
-            name = null;
-            add = false;
-        } else {
-            add = "add".equalsIgnoreCase(first);
-            name = Utils.addPackage(instruction.getPackage(), instruction.next());
+        final String action = instruction.next();
+        switch (action.toLowerCase(Locale.ROOT)) {
+            case "update":
+            case "refresh":
+            case "reload":
+                name = null;
+                add = false;
+                break;
+            case "add":
+                name = Utils.addPackage(instruction.getPackage(), instruction.next());
+                add = true;
+                break;
+            case "delete":
+            case "remove":
+            case "del":
+            case "rm":
+                name = Utils.addPackage(instruction.getPackage(), instruction.next());
+                add = false;
+                break;
+            default:
+                throw new InstructionParseException("Unknown journal action: " + action);
```

Comment:
We do not want to introduce new aliases.
For del we need maybe more aliases, because of the old code. But we should take one as recommended, and all other one should throw an warning about being deprecated.